### DISCUSSION TITLE
Changelog between 1.5.5 and 1.5.6

### DISCUSSION THREAD

**Discussion by MartyIX** [microsoft/microsoft-ui-xaml#9911] [Q&A] [answered]
Hi all,

I checked https://learn.microsoft.com/en-gb/windows/apps/windows-app-sdk/stable-channel#version-156-15240802000 and it contains just two bullet points. So I went into source to see what changed and I can see:

https://github.com/microsoft/microsoft-ui-xaml/compare/winui3/release/1.5.5...winui3/release/1.5.6

I must say I'm confused by so many changes. Could anyone explain the discrepancy?

Question number 2: Are the release notes https://learn.microsoft.com/en-gb/windows/apps/windows-app-sdk/stable-channel#version-156-15240802000 exhaustive? Or is it just "what customers need to know"?

Thank you in advance!

**Comment by codendone** [comment, marked as answer]
That diff seems to be showing the entirety of all files. My guess is the source drops are in a clean branch and the diff attempt doesn't work without a shared base. @evelynwu-msft ?

For question 2, that is the exhaustive release notes: just those two important fixes in 1.5.6.

**Comment by evelynwu-msft** [comment]
@MartyIX Thanks for bringing this to our attention! This was the result of an infrastructure issue on our part; we've manually fixed the 1.5.6 source drop (and 1.4.7 which we discovered was affected as well) and have plans to improve our tooling to avoid a repeat in the future.

**Reply by MartyIX** [reply]
Glad to hear that :-)
